toolip/utils/redoc.py

```diff
@@ -35,6 +35,18 @@ def make_docs_router(
     logo_alt_text: str,
     tags_and_models: List[TagAndModels],
 ) -> APIRouter:
+    """Creates a FastAPI docs router.
+
+    Args:
+        title: Title of the API documentation page.
+        description: A required str parameter.
```

Comment:
The user has to understand typing. Better to not duplicate info so no need to say "required str", the type hints will say so.
```suggestion
        description: Description section for the API documentation.
```